enhancements/ingress/optional-ingress-hypershift.md

```diff
@@ -0,0 +1,301 @@
+
+### Workflow Description
+
+The HyperShift hosted cluster's user wants to use an AWS ALB to route the traffic to its example application deployed on a HyperShift's hosted cluster.
+
+#### Terminology
+[HyperShift concepts and personas](https://hypershift-docs.netlify.app/reference/concepts-and-personas/).
+
+#### Disable the default ingress
+
+- Cluster service consumer requests a hosted cluster without the ingress capability using the HostedCluster API (**note**: the HostedCluster's capability API does not exist and it's out of scope of this enhancement).
+- HyperShift creates the hosted cluster.
+- HyperShift skips the deployment of the ingress operator on the hosted control plane.
+- HyperShift creates `ClusterVersion` without the ingress capability on the hosted cluster.
+- Hosted cluster's cluster version operator does not deploy the cluster ingress operator's payload (CRDs, RBACs, etc.).
+
+#### Setup AWS ALB as the new ingress
+
+- Cluster instance admin installs [AWS Load Balancer Operator](/enhancements/ingress/aws-load-balancer-operator.md).
+- Cluster instance admin sets up the default `AWSLoadBalancerController`.
+- Cluster instance user creates the `Ingress` resource for the default aws load balancer controller's ingress class.
+- AWS load balancer controller provisions an ALB on AWS.
+- Cluster instance admin can install [External DNS Operator](/enhancements/dns/externaldns-operator.md) to help cluster instance users set up convenient DNS names.
+
+### API Extensions
+
+N/A - no api extensions are being introduced in this proposal
+
+### Implementation Details/Notes/Constraints
+
+#### Ingress capability
+
+The name of the new capability is `Ingress`. It includes the cluster ingress operator and [all its APIs](#ingress-operator-apis) but does NOT include [the related APIs](#related-apis).
+The implementation of the new capability should follow the instructions of [how to implement a new capability](/enhancements/installer/component-selection.md#how-to-implement-a-new-capability), to name some:
+- `Ingress` capability should be added to `openshift/api` repository.
+- The ingress capability should be part of the `vCurrent` and `v4.14` baseline sets.
+- The openshift api vendored dependency should be bumped into `cluster-version-operator` repository.
+  - Additionally some more changes will need to be done in the cluster version operator to address [the standalone use case](#standalone-openshift).
+- The openshift api vendored dependency should be bumped in `installer` repository.
+  - This will allow the users of the standalone OpenShift to use the ingress capability. While the enabling of the ingress capability is not supposed to cause any issue, the disabling may have some consequences described in [Standalone OpenShift chapter](#standalone-openshift).
```

Comment:
```suggestion
  - This will allow the users of the standalone OpenShift to see the ingress capability though they will not be able to disable it(it will be implicitly enabled even if they don't choose to enable it). While the enabling of the ingress capability is not supposed to cause any issue, the disabling may have some consequences described in [Standalone OpenShift chapter](#standalone-openshift).
```

Reply:
Repharsed it differently to make the change minimal. I hope I didn't loose the essence.